metacat-main/src/main/java/com/netflix/metacat/main/configs/ServicesConfig.java

```diff
@@ -79,13 +83,50 @@ public UserMetadataService userMetadataService(
     }
 
     /**
+     * Tag service.
+     *
+     * @param dataSourceManager The datasource manager to use
+     * @param config            System config to use
+     * @param metacatJson       Json Utilities to use
+     * @return User metadata service based on MySql
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    public TagService tagService(
+        final DataSourceManager dataSourceManager,
+        final Config config,
+        final MetacatJson metacatJson
+    ) {
+        return new DefaultTagService();
+    }
+
+    /**
+     * Look up service.
+     *
+     * @param dataSourceManager The datasource manager to use
+     * @param config            System config to use
+     * @param metacatJson       Json Utilities to use
+     * @return User metadata service based on MySql
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    public LookupService lookupService(
+        final DataSourceManager dataSourceManager,
+        final Config config,
+        final MetacatJson metacatJson
+    ) {
+        return new DefaultLookupService();
+    }
+
+    /**
      * Lookup service.
      *
      * @param dataSourceManager Datasource manager to use
      * @param config            System configuration to use
      * @return Lookup service backed by MySQL
      */
     @Bean
+    @Profile({ProfileConstants.MYSQL_USERMETADATA})
```

Comment:
These beans should not be in this file they should be defined in a mysql implementation module. Lets talk about this in team meeting.